docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,11 @@
+# Only allow admins to create new streams
```

Comment:
This is in the wrong folder. That's why your Travis builds are failing. This file needs to be in the templates/zerver/help folder